### ISSUE TITLE
Disable updates check when launching the terminal

### ISSUE THREAD

**Issue by redochka** [scaleway/scaleway-cli#1899]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

When launching the terminal, it takes long time before it launches because scw is doing a request to check if a new release of scw is available. 

### How I imagine scw could expose this functionality

Please expose a way to disable this startup check
Also, maybe you can do this check at another time, maybe when the command has finish execution. Like `npm` does.

**Comment by pimterry** [comment]
Yes, please make the startup script silent & instant! I open & close terminals all day, I've just had to disable scw on my system because this is so annoying.

**Comment by remyleone** [comment]
I'm so sorry @pimterry we will start working on it very soon to make it available in a new version.
